src/test/java/com/google/plantasy/TimingPropertiesTest.java

```diff
@@ -50,61 +50,76 @@ public void getQuizTimestampProperty_nullUserTimeStamp() {
         Entity user = new Entity("user");
         user.setProperty("quiz_timestamp", null);
         user.setProperty("userID", "abc");
- 
+
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         datastore.put(user);
- 
-        Long actual = timing_properties_test.getQuizTimestampProperty("user", "userID", (user.getProperty("userID")).toString(), datastore); 
-        Assert.assertEquals(null, actual);
+
+        Long actual = null;
+        try {
+            actual = timing_properties_test.getQuizTimestampProperty("user", "userID", (user.getProperty("userID")).toString(), datastore); 
+        } catch (NullPointerException e) {
+            Assert.assertEquals(null, actual);
+        }
```

Comment:
here (and everywhere else in this PR) we'll want to actually assert that the exception was thrown. 

What you did is technically a correct way of accomplishing this goal, but it's not intuitive and there is a more elegant way to do this:

https://howtodoinjava.com/junit5/expected-exception-example/

Let me know if you have any questions!

Reply:
*insert thumbs up emoji*